nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -342,8 +351,15 @@ in {
 
   config = mkIf (config.boot.initrd.enable && cfg.enable) {
     system.build = { inherit initialRamdisk; };
+
+    boot.initrd.availableKernelModules = [ "autofs4" ]; # systemd needs this for some features
+
     boot.initrd.systemd = {
       initrdBin = [pkgs.bash pkgs.coreutils pkgs.kmod cfg.package] ++ config.system.fsPackages;
+      extraBin = {
```

Comment:
not really `extra`

Reply:
What else would you recommend here? `extraBinBinaries`?

Reply:
`extra` is generally for options that you can't `lib.mkForce []` because they start out empty and get added to something mandatory.

Reply:
Hmm but the thing is we already have `initrdBin` and are adding more tools to /bin :thinking: 

Maybe we should rename `initrdBin` as well